spec/server/functional/user.spec.coffee

```diff
@@ -234,6 +234,14 @@ ghlfarghlarghlfarghlarghlfarghlarghlfarghlarghlfarghlarghlfarghlarghlfarghlarghl
     expect(body.role).toBe('advisor')
     done()
 
+  it 'does not allow users to have no username or email', utils.wrap (done) ->
```

Comment:
A better name: `it 'returns 422 if both email and name would be unset for a registered user'`